scheduler/models.py

```diff
@@ -75,52 +78,39 @@ def clean_queue(self):
             })
 
     def is_scheduled(self):
-        return self.job_id and self.job_id in self.scheduler()
```

Comment:
This can return `None` if `self.job_id` is not set instead of just `True` or `False`.